sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -485,53 +485,52 @@ a!
 ----
 namespace sycl::ext::oneapi::experimental {
 
-struct registered_kernel_names {
-  registered_kernel_names();                                       (1)
-  registered_kernel_names(const std::string &name);                (2)
-  registered_kernel_names(const std::vector<std::string> &names);  (3)
-  void add(const std::string &name);                               (4)
+struct registered_names {
+  registered_names();                                       (1)
+  registered_names(const std::string &name);                (2)
+  registered_names(const std::vector<std::string> &names);  (3)
+  void add(const std::string &name);                        (4)
 };
-using registered_kernel_names_key = registered_kernel_names;
+using registered_names_key = registered_names;
 
 template<>
-struct is_property_key<registered_kernel_names_key> : std::true_type {};
+struct is_property_key<registered_names_key> : std::true_type {};
 
 } // namespace sycl::ext::oneapi::experimental
 ----
 !====
 
-This property is useful when the source language represents kernel names
-differently in the source code and the generated code.
+This property is useful when the source language represents names differently in
+the source code and the generated code.
 For example, {cpp} function names in the generated code are "mangled" in an
 implementation-defined way.
```

Comment:
Nit: It might be useful to give another example here of names that are mangled, to make it clear that this behavior is not unique to functions.  Something as simple as "{cpp} function names and the names of static variables at global scope", maybe?

Reply:
I like that; added.